app/preprints/-components/submit/author-assertions/public-preregistration/template.hbs

```diff
@@ -0,0 +1,72 @@
+<div local-class='form-container'>
+    <FormControls
+        @changeset={{@changeSet}}
+        data-test-form
+        as |form|
+    >
+        {{#let (unique-id 'publicPreregistration') as |publicPreregistrationId|}}
+            <label for={{publicPreregistrationId}}
+                data-test-public-preregistration-label
+            >
+                {{t 'preprints.submit.step-three.public-preregistration-input'}}
+                <span local-class='required'>*</span>
+            </label>
+            <p>
+                {{t 'preprints.submit.step-three.public-preregistration-description'}}
+            </p>
+
+            <form.radioGroup
+                @id={{publicPreregistrationId}}
+                @valuePath={{'hasPreregLinks'}}
+                @class='radio-group'
+                @isRequired={{true}}
+                @changeset={{@changeSet}}
+                @options={{this.publicPreregistrationOptions}}
+                @onchange={{this.updatePublicPreregistrationOptions}}
+                as |radioGroup|
+            >
+                {{radioGroup}}
+            </form.radioGroup>
+        {{/let}}
+
+        {{#if this.displayPublicPreregistrationLinks}}
+            <div local-class='input-container'>
+                <select
+                    onchange={{action 'updatePreregistrationLinkInfo' value='target.value'}}
+                    local-class='select'
+                    class='form-control'
+                >
+                    {{#each this.publicPreregLinkInfoOptions as |infoOption|}}
+                        <option value={{infoOption.key}}>
+                            {{infoOption.value}}
+                        </option>
+                    {{/each}}
+                </select>
+
+                <ValidationErrors
+                    local-class='validation-error'
+                    @changeset={{@changeSet}}
+                    @key={{'preregLinkInfo'}}
+                />
+            </div>
+            <div local-class='input-container'>
+                <Preprints::-Components::Submit::AuthorAssertions::DataLinkWidget
```

Comment:
Would it be too much trouble to change the `AuthorAssertions::DataLinkWidget` to just `AuthorAssertions::LinksWidget` since it's being used for DataLinks and PreregistrationLinks?